api-specs/Gateway-EE/3.5/kong-ee-3.5.yaml

```diff
@@ -10156,88 +11210,2472 @@ paths:
+                        id: 7574eb1d-c9fa-46a9-bd3a-3f1b4b196287
+                        name: read-only
+                        is_default: false
+                      - comment: 'Full access to all endpoints, across all workspaces—except RBAC Admin API'
+                        created_at: 1556563122
+                        id: 7fdea5c8-2bfa-4aa9-9c21-7bb9e607186d
+                        name: admin
+                        is_default: false
+                      - comment: 'Full access to all endpoints, across all workspaces'
+                        created_at: 1556563122
+                        id: 99bd8d18-f5b6-410e-aefe-d75f4252f13c
+                        name: super-admin
+                        is_default: false
+      tags:
+        - admins
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                Roles:
+                  type: string
+              x-examples:
+                Example 1:
+                  Roles: read-only
+        description: (comma separated) string of names of Roles to create or update for an Admin
+      description: Create or update roles for an admin
+    delete:
+      tags:
+        - admins
+      summary: Delete an Admin’s Role
+      operationId: delete-admins-name_or_id-roles
+      responses:
+        '204':
+          description: No Content
+      description: Delete an admin's roles by passing a comma seperated string of names of specific roles to remove from an admin.
+  '/admins/{name_or_id}/workspaces':
+    parameters:
+      - schema:
+          type: string
+        name: name_or_id
+        in: path
+        required: true
+        description: The Admin’s username or ID
+    get:
+      summary: List an Admin’s Workspaces
+      tags:
+        - admins
+      responses:
+        '200':
+          $ref: '#/components/responses/workspace-response'
+      operationId: get-admins-name_or_id-workspaces
+      description: Return workspaces associated with an admin.
+  '/debug/cluster/control-planes-nodes/log-level/{log_level}':
+    parameters:
+      - schema:
+          type: string
+          enum:
+            - debug
+            - info
+            - notice
+            - warn
+            - error
+            - crit
+        name: log_level
+        in: path
+        required: true
+        description: The log level
+    put:
+      summary: Set Node Log Level of All Control Plane Nodes
+      operationId: put-debug-cluster-control-planes-nodes-log-level-log_level
+      responses:
+        '200':
+          description: log level changed
+      description: |-
+        Change the log level of all Control Plane nodes deployed in Hybrid (CP/DP) cluster.
+        Care must be taken when changing the log level of a node to debug in a production environment because the disk could fill up quickly. As soon as the debug logging finishes, revert back to a higher level such as notice.
```

Comment:
```suggestion
        Be careful when changing the log level of a node to debug in a production environment because the disk could fill up quickly. As soon as the debug logging finishes, revert back to a higher level, such as notice.
```